src/Mod/PartDesign/Gui/TaskLoftParameters.cpp

```diff
@@ -250,8 +250,7 @@ void TaskLoftParameters::onDeleteSection()
         App::DocumentObject* obj = loft->getDocument()->getObject(data.constData());
         std::vector<App::DocumentObject*>::iterator f = std::find(refs.begin(), refs.end(), obj);
         if (f != refs.end()) {
-            refs.erase(f);
-            loft->Sections.setValues(refs);
+            loft->Sections.removeValue(obj);

```

Comment:
Can you please add a comment why you delete the object and not set the changed ref.

This is not straight forward and in two years... you know ;-)

Reply:
Force-pushed.

IMO though deleting individual object sounds like the more intuitive way. If we have a method, why not use it?